flyteplugins/go/tasks/pluginmachinery/flytek8s/k8s_resource_adds.go

```diff
@@ -17,6 +17,9 @@ import (
 
 const (
 	flyteExecutionURL = "FLYTE_EXECUTION_URL"
+
+	FlyteInternalWorkerNameEnvVarKey        = "_F_WN"  // "FLYTE_INTERNAL_WORKER_NAME"
```

Comment:
```suggestion
	flyteInternalWorkerNameEnvVarKey        = "_F_WN"  // "FLYTE_INTERNAL_WORKER_NAME"
```

Reply:
Nit: the var above (and most others in flyte) uses lower camel case instead of pascal case.

Reply:
This can't be done, right? These are used by the plugin, which is in a different package.

Reply:
TIL that capitalization controls visibility of variables in go, never mind, sorry.